ops/packer/build_bfd-all.json

```diff
@@ -69,6 +69,7 @@
     {
       "only": ["bfd-pipeline"],
       "type": "ansible",
+      "user": "ec2-user",
```

Comment:
Due to differences in the default container user, I had to set this to get Ansible working.  I thought Jenkins ran as the `jenkins` user, so I"m not sure how this ever worked.  🤷